ArduPlane/quadplane.cpp

```diff
@@ -2501,7 +2501,11 @@ void QuadPlane::vtol_position_controller(void)
         const float distance = diff_wp.length();
         const Vector2f rel_groundspeed_vector = landing_closing_velocity();
         const float rel_groundspeed_sq = rel_groundspeed_vector.length_squared();
-        const float closing_groundspeed = rel_groundspeed_vector * diff_wp.normalized();
+        float closing_groundspeed = 0;
+
+        if (is_positive(distance)) {
```

Comment:
We use a threshold of 0.1 later in this function instead of just "is_positive".  That might be to stop the vector jumping around too much.

Reply:
yep, good point

Reply:
@mattbooker are you OK to change this to `>0.1`?  I'm happy to do it myself if you'd like.

Reply:
My bad, slipped my mind. I got it.